src/Features/User/Components/Profile.js

```diff
@@ -9,13 +9,17 @@ import ProfileEditModal from "./ProfileEditModal";
 const Profile = () => {
   const [showModal, setShowModal] = useState(false);
   const dispatch = useDispatch();
-  const { username, email } = useSelector((store) => store.auth);
+  const { username, email, following, followers } = useSelector(
+    (store) => store.auth
+  );
   const { posts, status } = useSelector((store) => store.posts);
 
   useEffect(() => {
     dispatch(getUsersPosts({ username }));
   }, [dispatch, username]);
 
+  console.log(followers);
```

Comment:
You can remove this console

Reply:
You can check , by liking one post and keep another as it it is. Its working.